### ISSUE TITLE
Add photos illustrating the brand in use

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/brand#10]
[See action shots of @bvhme here](https://drive.google.com/drive/folders/1S56DnyQz0tlnV9k-cqpU8Ea7IXkOO3Rd).

**Comment by Ainali** [comment]
Yes, good idea. We could also take a few photos from our door signs, our new sweater and the governance game.
